pycbc/inference/likelihood.py

```diff
@@ -718,6 +720,19 @@ def loglr(self, **params):
                 # if the waveform terminates before the filtering low frequency
                 # cutoff, there is nothing to filter, so just go onto the next
                 continue
+            # recalibrate the data if calibration options are present
+            if self.recal:
+                # get cal params
+
+                # adjust strain using cal params and model
+                h = recal[det].adjust_strain(h, delta_fs=delta_fs,
+                                             delta_qinv=delta_qinv,
+                                             delta_fc=delta_fc,
+                                             kappa_c=kappa_c,
+                                             kappa_tst_re=kappa_tst_re,
+                                             kappa_tst_im=kappa_tst_im,
+                                             kappa_pu_re=kappa_pu_re,
+                                             kappa_pu_im=kappa_pu_im)
```

Comment:
@dfinstad I know this is still a work in progress, but one early comment: the likelihood class should remain agnostic to the details of the recalibration. So, I'd add a function to the Recalibration class that can handle being given a dictionary that has additional parameters it doesn't care about in it. That function would pull out the things it needs to pass on to `adjust_strain`. Then you can just pass `params` here.